yaksh/static/yaksh/js/bootstrap-tabs.js

```diff
@@ -0,0 +1,80 @@
+/* ========================================================
+ * bootstrap-tabs.js v1.4.0
+ * http://twitter.github.com/bootstrap/javascript.html#tabs
```

Comment:
We do not need this additional JS!
Tabs should work with the updated bootstrap JS and CS.
You need to update the HTML template along with custom JS and CSS (for design question paper) to work with the updated bootstrap.